arrow-syntax.api

```diff
@@ -0,0 +1,1632 @@
+	public static synthetic fun invoke$default (Lkotlin/jvm/functions/Function16;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Ljava/lang/Object;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;ILjava/lang/Object;)Lkotlin/jvm/functions/Function15;
+	public static synthetic fun invoke$default (Lkotlin/jvm/functions/Function16;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Ljava/lang/Object;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;ILjava/lang/Object;)Lkotlin/jvm/functions/Function15;
+	public static synthetic fun invoke$default (Lkotlin/jvm/functions/Function16;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Ljava/lang/Object;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;ILjava/lang/Object;)Lkotlin/jvm/functions/Function15;
+	public static synthetic fun invoke$default (Lkotlin/jvm/functions/Function16;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Ljava/lang/Object;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;ILjava/lang/Object;)Lkotlin/jvm/functions/Function15;
+	public static synthetic fun invoke$default (Lkotlin/jvm/functions/Function16;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Larrow/syntax/function/Partial;Ljava/l
```

Comment:
I think this is something we can perhaps cut to more than half and not support so many arities as arriving at this place is extremely obscure and rare.

Reply:
Already deprecated